qcut/docs/issues/large-files-analysis/REFACTOR-AI-CONSTANTS.md

```
@@ -0,0 +1,547 @@
+// ID aliases for backward compatibility
+export const T2V_MODEL_ID_ALIASES: Record<string, T2VModelId> = { ... };
+
+// Capabilities per model (UI-driven config)
+export const T2V_MODEL_CAPABILITIES: Record<T2VModelId, T2VModelCapabilities> = { ... };
+
+// Helper to compute combined capabilities
+export function getCombinedCapabilities(selectedModelIds: T2VModelId[]): T2VModelCapabilities;
+```
+
+### Pattern 4: `ai-video/` directory structure
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)